src/gsi/read_gfs_ozone_for_regional.f90

```diff
@@ -335,6 +425,29 @@ subroutine read_gfs_ozone_for_regional
         write(6,*)'READ_GFS_OZONE_FOR_REGIONAL:  ***ERROR*** INVALID value for nvcoord=',sighead%nvcoord,filename
         call stop2(85)
      endif
+   else if ( use_gfs_ncio ) then !LB netCDF option
+     if (gfshead%nvcoord == 1) then
+        write(6,*) 'nvcord test1'
```

Comment:
Remove debug prints here and below, remove initials from above line comment.

Reply:
Removed.